documentation/sphinx/source/command-line-interface.rst

```diff
@@ -485,6 +485,8 @@ Lists the tenants present in the cluster.
 
 ``LIMIT`` - the number of tenants to list. Defaults to 100.
 
+``OFFSET`` - the number of items to skip over, starting from the beginning of the range. Defaults to 0.
```

Comment:
There's not anything wrong with having this parameter, but I think any user of this would be better served by specifying an updated begin key rather than specifying an offset just because the latter is going to get increasingly expensive when there are a lot of tenants.

Reply:
Yes, I agree, but I think Jan's use case needs pagination/offsets based on what we discussed.